src/docs/Ways of Working/git.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 4
+slug: /git-flow
+---
+
+# Git flow
```

Comment:
Git Flow and Github Flow are slightly different. As you mention further on, we're using the simpler 'Github workflow' 👍